src/pipx/main.py

```diff
@@ -350,6 +351,14 @@ def _add_install(subparsers: argparse._SubParsersAction) -> None:
             f"Requires Python {MINIMUM_PYTHON_VERSION} or above."
         ),
     )
+    p.add_argument(
+        "--preinstall",
+        action="append",
```

Comment:
Sounds good!

Reply:
Shouldn’t it be `+`? But in either case how do you provide a positional argument (i.e. name of the main package) after `--preinstall`?

Reply:
You can use `--` to stop absorbing the arguments.

Reply:
Since injecting one single package is likely the overwhelmingly majority of use cases, I feel we should stick to the current parser config (_not_ using `nargs="*"`) since that yields a simpler syntax most of the time.